src/buttonSection.js

```diff
@@ -0,0 +1,13 @@
+export default function () {
+  const btnContainer = document.createElement("div");
+  btnContainer.classList.add("button-container");
+  const homeBtn = document.createElement("button");
```

Comment:
add a line between line 3 and 4 and after 7th line as well, ideally you should add line break for separating logical sections of your functions.
For ex: here you are doing something with btnContainer in line 2-3 and from line 4 to line 7 you are creating home button, so ideally there should be a line break so that as a reader i can see that you are doing multiple things in your code.

Reply:
Do this line breaking in rest of the code as well